config.py

```diff
@@ -1,9 +1,19 @@
 import os
+from pathlib import Path
 
 debug = os.getenv("NWS_DROUGHT_DEBUG", False)
 DEBUG_MODE = False if (debug is False or debug == "False") else True
 dl_loc = os.getenv("NWS_DROUGHT_DOWNLOAD_DIR", False)
-DOWNLOAD_DIR = dl_loc if dl_loc is not False else "/tmp/nws_drought/"
+
+DOWNLOAD_DIR = Path(dl_loc if dl_loc is not False else "/tmp/nws_drought/")
+DOWNLOAD_DIR.mkdir(exist_ok=True, parents=True)
+input_loc = os.getenv("NWS_DROUGHT_INPUTS_DIR", False)
```

Comment:
I suggest setting `input_loc` to the default location `/workspace/Shared/Tech_Projects/NWS_Drought_Indicators/project_data/climatologies` if `NWS_DROUGHT_INPUTS_DIR` is not set.